conformance/setup.go

```diff
@@ -62,7 +66,6 @@ const (
 	envVarPush                      = "OCI_TEST_PUSH"
 	envVarContentDiscovery          = "OCI_TEST_CONTENT_DISCOVERY"
 	envVarContentManagement         = "OCI_TEST_CONTENT_MANAGEMENT"
-	envVarPushEmptyLayer            = "OCI_SKIP_EMPTY_LAYER_PUSH_TEST"
```

Comment:
this env var is mentioned in the conformance/README.md. The language there should be modified to reflect the new state